spring-native/storage-blob/README.md

```diff
@@ -0,0 +1,255 @@
+# Using Spring Cloud Azure Storage Blob Starter
+
+This code sample demonstrates how to read and write files with the [Spring Resource](https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#resources) abstraction for Azure Storage using the Spring Cloud Azure storage starter.
+
+## What You Will Build
+You will build an application that use Spring Resource abstraction to read and write data with [Azure Storage Blob](https://azure.microsoft.com/services/storage/blobs/).
+
+## What You Need
+
+- [An Azure subscription](https://azure.microsoft.com/free/)
+- [Terraform](https://www.terraform.io/)
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [JDK8](https://www.oracle.com/java/technologies/downloads/) or later
```

Comment:
Is this correct?

Reply:
I have added the `System Requirements` for these two building ways in each section, let me add docker and native-image here.